discovery-provider/plugins/network-monitoring/docker-compose.yaml

```diff
@@ -0,0 +1,74 @@
+
+version: '3'
+services:
+  network-monitoring-db:
```

Comment:
Probably but I wasn't sure exactly how we wanted to do it because in theory, they don't _have_ to be the same postgres instance. I can get rid of it now though

Reply:
Adding a card for this